qiskit/providers/models/backendconfiguration.py

```diff
@@ -38,6 +39,14 @@ class GateConfig:
                   and CX.
     """
 
+    @deprecate_func(
+        since="1.2",
+        removal_timeline="in the 2.0 release",
+        additional_msg="This class is not necessary for BackendV2. If user still need Qobj, "
+        "that probably means that they are using a backend based on the "
+        "deprecated BackendV1 class.",
+        stacklevel=3,
```

Comment:
what about this message for all the classes in the file?

```suggestion
        additional_msg="The models in ``qiskit.providers.models`` are part "
        "of the deprecated `BackendV1` workflow  and no longer necessary for `BackendV2`. If a user "
        "workflow requires these representations it likely relies on deprecated functionality and "
        "should be updated to use `BackendV2`.",
        stacklevel=3,
```